pkg/operator/ceph/file/spec.go

```diff
@@ -29,6 +29,8 @@ import (
 
 const (
 	mdsDaemonCommand = "ceph-mds"
+	// MDS cache memory limit should be set to 50-60% of RAM reserved for the MDS container
```

Comment:
Can we also have some more info about why this is necessary? Link to docs or some short rationale?

Reply:
@batrick?

Reply:
MDS uses approximately 125% of the value of `mds_cache_memory_limit` in RAM. Eventually we will tune this automatically: http://tracker.ceph.com/issues/36663

That will eventually be backported to Nautilus.